libavformat/dashdec.c

```diff
@@ -2017,7 +2018,11 @@ static void delay_reload(DASHContext *c, int64_t reload_count)
 
     // Attempt first reload immediately, otherwise wait the reload
     if (reload_count > 1) {
-        delay = c->reload_retry_interval * 1000;
+        if (c->reload_retry_interval > 0) {
```

Comment:
I still think you want to set the default dictionary option value to 0 instead of -1 to avoid ffmpeg complaining

Reply:
no harm in it anyways.